### PR TITLE
Add warning about executeCommand/open files

### PR DESCRIPTION
**Pull request by kevinquinnyo**
This is the same as https://github.com/cakephp/docs/pull/7304 but for 3.x as it applies here as well.

### CODE REVIEW

File: en/console-and-shells/commands.rst [hunk 1]
```diff
@@ -244,6 +244,14 @@ You may need to call other commands from your command. You can use
 .. versionadded:: 3.8.0
     ``executeCommand()`` was added.
 
+
+.. note::
+
+    When calling ``executeCommand`` in a loop, it is recommended to pass in the
```
Comment (dereuromark) [inline en/console-and-shells/commands.rst]:
```suggestion
    When calling ``executeCommand()`` in a loop, it is recommended to pass in the
```
maybe to clarify the method

Comment (kevinquinnyo) [reply en/console-and-shells/commands.rst]: Happy to make that change, but do you think I should also apply that to the corresponding 4.x change that was already merged?  This was basically a copy/paste of that one

Comment (othercorey) [reply en/console-and-shells/commands.rst]: Yes, make the change in both, please.

Comment (kevinquinnyo) [reply en/console-and-shells/commands.rst]: done, and here is corresponding 4.x: https://github.com/cakephp/docs/pull/7309

**Pull request merged**